backend/WikiContrib/WikiContrib/settings.py

```diff
@@ -152,7 +152,24 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'uploads')
 
 API_TOKEN = env('PHAB_KEY')
+GITHUB_ACCESS_TOKEN = env('GITHUB_ACCESS_TOKEN')
 CORS_ORIGIN_ALLOW_ALL = True
+GITHUB_FALLBACK_TO_PR = False
+githubAPILimit = 30
```

Comment:
I think the naming style for this variable should be like others `GITHUB_API_LIMIT`

Reply:
Okkkkk. I will fix that as well